subworkflows/local/input_check.nf

```diff
@@ -9,55 +9,78 @@ workflow INPUT_CHECK {
     samplesheet // file: /path/to/samplesheet.csv
 
     main:
-    parsed_samplesheet = SAMPLESHEET_CHECK ( samplesheet )
+
+    // Table to list, group per sample, detect if sample has multi-run,
+    // then spread back to per-run rows but with multi-run info added to meta
+    ch_split_samplesheet = SAMPLESHEET_CHECK ( samplesheet )
```

Comment:
You still have the SAMPLESHEET_CHECK modules with the nf-validation plugin?

Reply:
Yes, that will be updated in the next release. Please see https://github.com/nf-core/taxprofiler/issues/336